config/members-crossplane.yaml

```diff
@@ -0,0 +1,263 @@
+apiVersion: organizations.github.crossplane.io/v1alpha1
+kind: Membership
+metadata:
+  name: bassam-crossplane
```

Comment:
Would it make sense for these to read least-to-most-specific - e.g. `crossplane-bassam`? Might also be handy to label them by org so we can easily `kubectl get membership -l org=crossplane`.

Reply:
Great idea! I imagine as provider-github continues to mature we will have some XRDs that make these a little less unwieldy by doing things such as patching labels with org name, etc. -- but manually doing for now is a good idea 👍